### ISSUE TITLE
v1.3.5-1 on ubuntu bionic

### ISSUE THREAD

**Issue by tommyp-vr4** [gerbera/gerbera#541]
I've updated my ubuntu bionic today with the 1.3.5-1.  Seems I can't remove database items from the browser (firefox v71).  Is this a known issue or a snafu?  Don't mind providing more inputs, just let me know how or what data I can provide.  In the meantime, I will be going back to the previous version that worked fine.  (Seems this 1.3.5-1 version is a bit resource hungry too.)
Thanks
Tom

**Comment by whyman** [comment]
Hi @tommyp-vr4, can you post  a debug log?

**Comment by tommyp-vr4** [comment]
Well, good news and bad news.  Firstoff, some background info - my gerbera runs headless.  Bad news is that I could not figure out how to run gerbera with the gerbera account.  But the good news is that I may have stumbled on the problem - the old database and new database aren't compatible.  Maybe someone else can confirm?  
Here's some investigating I did:
1. Launch new gerbera using the old database.  Can't add or remove anything from database via network machine browser.  Loggin in via SSH tells me that the "system information is disabled due to load higher than 2.0."  I never saw that via SSH before!  I "stopped" gerbera, exited SSH and logged back in via SSH and no weird "load" error.  This tells me that gerbera has something wrong with it.

2. Launch old gerbera with old database.  No "load" errors via ssh.  I can add/remove database entries via network machine browser.  This tells me that old gerbera is OK.

3.  Via ssh, I install the new gerbera and issue a service gerbera stop command.  As root, I deleted the /var/lib/gerbera.db.  To be conservative, I rebooted the machine, logged back in via SSH and no "load" errors.  I was then able to add stuff to the library via networked machine without issue.  So it's all systems go for me now.  (It can be noted that creating a new library database does make the machine "work" harder but system is normal after database is done being built.)

Maybe I did something weird, I have no idea, but my issue is closed.  Might be a good idea to flag users that they may need to stop gerbera, manually delete the database file and restart gerbera if they cannot add/remove anything from a browser.
